### DISCUSSION TITLE
how to code spritepacks?? pls do it for me

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#7941] [general - spritepacks]
hi, so, i wanna give monika the spritepack i made, but i cannot. i do not know how to code json and idk how to make gift reactions either. HELP!! btw, ive tried the wiki's tutorial on it but its confusing. can someone code it for me?..here is the spritepack to code the json and the gift thingy..

[pink_gown 4.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7217579/pink_gown.4.zip)

**Comment by username11421** [comment]
For writing the gift reaction, see the zz_reactions.rpy file. Just copy the code format of one of the other functions.

**Comment by qiqsom** [comment]
I'm not sure if it's going to work on MAC but try it here: 
[pink_gown.zip](https://github.com/Monika-After-Story/MonikaModDev/files/7221890/pink_gown.zip)
By the way, I didn't make a custom reaction lol, sorry.
